lib/meilisearch/pagination.rb

```diff
@@ -7,12 +7,11 @@ def self.create(results, total_hits, options = {})
       return results if MeiliSearch.configuration[:pagination_backend].nil?
 
       begin
-        backend = MeiliSearch.configuration[:pagination_backend].to_s.gsub(/\/(.?)/) { "::#{$1.upcase}" }.gsub(/(?:^|_)(.)/) { $1.upcase } # classify pagination backend name
-        page = Object.const_get(:MeiliSearch).const_get(:Pagination).const_get(backend).create(results, total_hits, options)
```

Comment:
As the idea is to just classify the configuration to be used as a class, we could use the [classify](https://apidock.com/rails/ActiveSupport/Inflector/classify) method in order to get the same behavior :)